src/components/ProteinViewer/ShowMoreTracks/index.tsx

```diff
@@ -20,44 +19,47 @@ type Props = {
     name: string[],
     hide: boolean,
   ) => CategoryVisibility;
+  changeSettingsRaw: (
+    category: string,
+    key: string,
+    value: string | number | boolean | LabelUISettings,
+  ) => SettingsAction;
 };
 
-{
-  /* Button to show/hide all secondary tracks (maintains internal view/show state of the specific track)*/
-}
 const ShowMoreTracks = ({
   showMore,
   hideCategory,
   showMoreChanged,
   setHideCategory,
   switchCategoryVisibilityShowMore,
+  changeSettingsRaw, // Destructure the prop
 }: Props) => {
+  const handleClick = () => {
+    // Update the hideCategory state
+    const newHideCategory = switchCategoryVisibilityShowMore(
+      hideCategory,
+      ['families', 'domains'],
+      !showMore ? false : true,
+    );
+    setHideCategory(newHideCategory);
+    showMoreChanged(!showMore);
+
+    // Save the showMore state to the Redux store
+    changeSettingsRaw('ui', 'showMoreSettings', !showMore);
+  };
+
   return (
-    <div>
-      <button
-        style={{
-          boxShadow: 'none',
-          transform: 'none',
-        }}
-        className={css(
-          'vf-button',
-          'vf-button--secondary',
-          'vf-button--sm',
-          'showmore-btn',
-        )}
-        onClick={() => {
-          setHideCategory(
-            switchCategoryVisibilityShowMore(
-              hideCategory,
-              ['families', 'domains'],
-              !showMore ? false : true,
-            ),
-          );
-          showMoreChanged(!showMore);
-        }}
+    <div onClick={handleClick}>
+      <div
+        className={css('showmore-toggle-outer', showMore ? 'full' : 'overview')}
       >
-        {showMore ? 'Show summary view' : 'Show all annotations'}
-      </button>
+        <div
+          className={css('showmore-toggle-inner', showMore ? 'moved' : '')}
+        />
+        <span style={{ marginLeft: showMore ? '7px' : '25px' }}>
+          {showMore ? 'Full View' : 'Overview'}
+        </span>
+      </div>
     </div>
   );
```

Comment:
Minor style changes, and add a label in front of the switch (not the best UI, but I think we need a label):

```ts
  return (
    <>
      Feature display:
      <div onClick={handleClick}>
        <div
          className={css('showmore-toggle-outer', showMore ? 'full' : 'overview')}
        >
          <div
            className={css('showmore-toggle-inner', showMore ? 'moved' : '')}
          />
          <span style={{ marginLeft: showMore ? '5px' : '33px' }}>
          {showMore ? 'Full' : 'Overview'}
        </span>
        </div>
      </div>
    </>
  );
```